src/mcp_agent/cli/cloud/commands/logger/tail/main.py

```
@@ -129,14 +129,17 @@ def tail_logs(
         console.print("[red]Error: --format must be 'text', 'json', or 'yaml'[/red]")
         raise typer.Exit(6)
     
-    app_id, config_id, server_url = parse_app_identifier(app_identifier)
+    try:
+        app_id, config_id = parse_app_identifier(app_identifier)
+    except ValueError as e:
+        console.print(f"[red]Error: {e}[/red]")
```

Should be print_error. I think the root-level wrapper would also already handle any CLIError thrown instead of us explicitly printing error and exiting